sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/core/generator/AbstractQueryGenerator.java

```diff
@@ -255,7 +258,7 @@ static String generateQuerySort(@NonNull Sort sort) {
     private String generateQueryTail(@NonNull CosmosQuery query) {
         final List<String> queryTails = new ArrayList<>();
 
-        queryTails.add(generateQuerySort(query.getSort()));
+        queryTails.add(generateQuerySort(query.getSort(), "r"));
```

Comment:
why not use the same table name constant variable here?

Reply:
It wasn't a constant, it was the tableName to use when generating the sort query. However, with the new constant I am using it here now.